pkg/cdi/version.go

```diff
@@ -115,6 +117,23 @@ func (r requiredVersionMap) requiredVersion(spec *cdi.Spec) version {
 	return minVersion
 }
 
+// requiresV060 returns true if the spec uses v0.6.0 features
+func requiresV060(spec *cdi.Spec) bool {
+	// The v0.6.0 spec allows annotations to be specified at a spec level
+	for range spec.Annotations {
+		return true
+	}
+
+	// The v0.6.0 spec allows annotations to be specified at a device level
+	for _, d := range spec.Devices {
+		for range d.Annotations {
+			return true
```

Comment:
Do we need a loop here if we unconditionally return on the first iteration?

Reply:
The intent of the loop is to detect whether a `v0.6.0` spec is required. This is required if ANY annotions are detected. 

We could use:
```
	for _, d := range spec.Devices {
   		if len(d.Annotations) > 0 {
			return true
```
instead, but I thought the loop was cleaner than the conditional in this case.

Reply:
Not for me, but I'm ok with it.